app/messages/send-notification.js

```diff
@@ -22,9 +22,9 @@ const trySendViaNotify = async (message, emailAddress) => {
 }
 
 const sendNotification = async (message) => {
-  const emailAddresses = Array.isArray(message.body.data.commsAddress)
-    ? message.body.data.commsAddress
-    : [message.body.data.commsAddress]
+  const emailAddresses = Array.isArray(message.body.data.commsAddresses)
+    ? message.body.data.commsAddresses
+    : [message.body.data.commsAddresses]
 
```

Comment:
It was suggested during a dev call to update `commsAddress` to instead be `commsAddresses` for clarity so that teams using SFD to send messages know they can provide more than one email per message sent.